icu4j/main/classes/core/src/com/ibm/icu/impl/breakiter/CjkBreakEngine.java

```diff
@@ -14,25 +14,68 @@
 
 import java.io.IOException;
 import java.text.CharacterIterator;
+import java.util.HashSet;
 
 import com.ibm.icu.impl.Assert;
+import com.ibm.icu.impl.ICUData;
 import com.ibm.icu.text.Normalizer;
 import com.ibm.icu.text.UnicodeSet;
+import com.ibm.icu.text.UnicodeSetIterator;
+import com.ibm.icu.util.UResourceBundle;
+import com.ibm.icu.util.UResourceBundleIterator;
 
 public class CjkBreakEngine extends DictionaryBreakEngine {
     private UnicodeSet fHangulWordSet;
+    private UnicodeSet fNumberWordSet;
```

Comment:
so fNumberWordSet and fOpenPunctuationdSet are use exactly the same place together. Could we just use one UnicodeSet for that? 
fNumberOrOpenPunctuationdSet = new UnicodeSet("[[:Nd:][:Pi:][:Ps:]]");

Reply:
Resolved